packages/web-app-files/tests/unit/components/AppBar/CreateAndUpload.spec.ts

```diff
@@ -173,10 +171,25 @@ function getWrapper({
     mock<FileAction>({ label: () => 'Draw.io document', ext: 'drawio' })
   ]
 } = {}) {
+  const capabilities = {
+    spaces: { enabled: true },
+    files: { app_providers: [{ new_url: '/' }] }
+  } satisfies Partial<CapabilityStore['capabilities']>
+
+  const plugins = defaultPlugins({
+    piniaOptions: {
+      spacesState: { spaces },
+      capabilityState: { capabilities },
+      clipboardState: { resources: clipboardResources },
+      resourcesStore: { areFileExtensionsShown, currentFolder, resources: files }
+    }
+  })
+
   vi.mocked(useRequest).mockImplementation(() => ({
     makeRequest: vi.fn().mockResolvedValue({ status: 200 })
   }))
-  vi.mocked(useExtensionRegistry).mockImplementation(() => useExtensionRegistryMock())
+  const { requestExtensions } = useExtensionRegistry()
+  vi.mocked(requestExtensions).mockReturnValue([])
```

Comment:
Wondering if this is really needed. Since the default `extensions` state should be `[]`, I'd expect `requestExtensions` to return the same.

Reply:
I can set `stubActions: false`, then your statement is true. Otherwise `requestExtensions` returns `undefined`. Which one do you prefer?

Reply:
I always forget that this is the default 😄 Then it's fine like this IMO.